feature-libs/cart/quick-order/components/quick-order/quick-order.component.ts

```diff
@@ -9,21 +14,26 @@ import {
   OrderEntry,
 } from '@spartacus/core';
 import { CmsComponentData } from '@spartacus/storefront';
-import { Observable } from 'rxjs';
+import { combineLatest, Observable, Subscription } from 'rxjs';
 import { filter, first, map, switchMap } from 'rxjs/operators';
 
 @Component({
   selector: 'cx-quick-order',
   templateUrl: './quick-order.component.html',
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class QuickOrderComponent implements OnInit {
+export class QuickOrderComponent implements OnInit, OnDestroy {
   cartId$: Observable<string>;
   entries$: Observable<OrderEntry[]>;
-  isCartLoading$: Observable<boolean> = this.activeCartService.isStable();
   quickOrderListLimit$: Observable<
     number | undefined
   > = this.component.data$.pipe(map((data) => data.quickOrderListLimit));
+  isCartStable$: Observable<any> = combineLatest([
```

Comment:
```suggestion
  isCartStable$: Observable<boolean> = combineLatest([
```

However, let's talk about this, or maybe you can bring this up to someone as I know GC will complain once he sees this flicker.

Example of the flicker @ 19s and 34s


https://user-images.githubusercontent.com/29099466/127596047-30c191ef-d7f2-4bfc-b620-7059ed9bd590.mov

Reply:
You can try hooknig into the `CartAddEntryEvent` for load and remove the spinner when `CartEntryAddEntrySuccess/FailEvent` is fired